src/main/resources/camunda/evidence_upload_scheduler.bpmn

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_02cs6jr" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.3.0">
+  <bpmn:process id="EVIDENCE_UPLOAD_SCHEDULER" isExecutable="true">
+    <bpmn:sequenceFlow id="Flow_03at42s" sourceRef="StartEvent_1" targetRef="EVIDENCE_UPLOAD_CHECK" />
+    <bpmn:sequenceFlow id="Flow_1m5i8op" sourceRef="EVIDENCE_UPLOAD_CHECK" targetRef="Event_1ifnpoe" />
+    <bpmn:serviceTask id="EVIDENCE_UPLOAD_CHECK" name="Evidence Upload scheduler" camunda:type="external" camunda:topic="EVIDENCE_UPLOAD_CHECK">
+      <bpmn:incoming>Flow_03at42s</bpmn:incoming>
+      <bpmn:outgoing>Flow_1m5i8op</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:startEvent id="StartEvent_1">
+      <bpmn:outgoing>Flow_03at42s</bpmn:outgoing>
+      <bpmn:timerEventDefinition id="TimerEventDefinition_1oppfcm">
+        <bpmn:timeCycle xsi:type="bpmn:tFormalExpression">0 0 18 * * ?</bpmn:timeCycle>
```

Comment:
Have a look at TakeCaseOfflineScheduler + CaseDismissed scheduler.
They seem to run on the assumption that the working day stops at 4, and they run 4:05.
Did you discuss with someone that it should be 6?

Reply:
yes.. Doug asked me to put it at 6.

Reply:
Ok, no problem :)